roles/edpm_bootstrap/defaults/main.yml

```diff
@@ -49,3 +49,6 @@ edpm_bootstrap_network_service: NetworkManager
 
 # String for SELinux state. One of: disabled, enforcing, permissive
 edpm_bootstrap_selinux_mode: enforcing
+
+# KSM control
+edpm_bootstrap_enable_ksm: false
```

Comment:
hum ok i was expecting this to be in its own role.

i guess bootstrap si not terrible but lets see what others think

Reply:
+1 to creating a seprate role (edpm_ksm ?). We can include it from bootstrap but adding more service logics directly to bootstrap playbook makes future maintenance hard.

Reply:
Why not the edpm_kernel role in a ksm.yml tasks file?

Reply:
I would also argue for keeping this functionality separate. However, we could implement this as it's own role, and invoke that, conditionally, from edpm_bootstrap. 

How about that?

Reply:
I would prefer @slagle's approach. No need for a dedicated role for such a tiny config bit?